app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    file_name = command.split()[1]
+    new_path = command.split()[2]
+    path_list = new_path.split("/")
+
+    if command.split()[0] == "mv":
+        if len(path_list) > 1:
+            new_dir = os.path.dirname(new_path)
+            if not os.path.exists(new_dir):
+                os.makedirs(new_dir)
```

Comment:
Be cautious when using `os.makedirs` to create directories, as attempting to create a directory that already exists will lead to an error. Consider using `os.makedirs(new_dir, exist_ok=True)` to avoid this issue, as per the checklist item: 'Notice that if you will try to create the directory already exists then it will lead to an error.'